clap.zig

```diff
@@ -1390,16 +1391,17 @@ pub fn help(
     var first_parameter: bool = true;
     for (params) |param| {
         if (!first_parameter)
-            try writer.writeByteNTimes('\n', opt.spacing_between_parameters);
+            try writer.splatByteAll('\n', opt.spacing_between_parameters);
 
         first_parameter = false;
-        try writer.writeByteNTimes(' ', opt.indent);
+        try writer.splatByteAll(' ', opt.indent);
 
-        var cw = ccw.codepointCountingWriter(writer);
-        try printParam(cw.writer(), Id, param);
+        var ccw_buffer: [256]u8 = undefined;
+        var cw = ccw.CodepointCountingWriter.init(writer, &ccw_buffer);
```

Comment:
Do we really need a buffer here? If the user wants buffering, they should pass a writer that is buffered. The CodepointCountingWriter can just be unbuffered